develop/tutorials/articles/192-item-selector/01-selecting-entities-using-the-item-selector-api.markdown

```diff
@@ -0,0 +1,356 @@
+    can be created by invoking with either `HttpServletRequest` or
+    `PortletRequest`. Use the following pattern to create your URL:
+    `RequestBackedPortletURLFactoryUtil.create(request)`.
+- `ItemSelectedEventName`: a unique JavaScript event name that is
+    triggered by the item selector when the element is selected.
+- `ItemSelectorCriterion`: the criterion (or array of criteria) that
+    specifies the element that should be selected.
+    
+Follow the pattern below to obtain the URL for the Item Selector:
+
+    List<ItemSelectorReturnType> desiredItemSelectorReturnTypes =  new
+    ArrayList<>();
+    
+    desiredItemSelectorReturnTypes.add(new *ItemSelectorReturnType());
+    
+    *ItemSelectorCriterion class *nameItemSelectorCriterion =   new
+    *ItemSelectorCriterion();
+    
+    *nameItemSelectorCriterion.setDesiredItemSelectorReturnTypes(
+    desiredItemSelectorReturnTypes);
+    
+    PortletURL itemSelectorURL = `_itemSelector.getItemSelectorURL`(
+    
+      requestBackedPortletURLFactory, "ItemSelectedEventName",
+    
+      *ItemSelectorCriterion);
+
+For example, the following configuration obtains the item selector URL for the
+`ImageItemSelectorCriterion`:
+
+    List<ItemSelectorReturnType> desiredItemSelectorReturnTypes =  new
+    ArrayList<>();
+    
+    desiredItemSelectorReturnTypes.add(new URLItemSelectorReturnType());
+    
+    ImageItemSelectorCriterion imageItemSelectorCriterion =   new
+    ImageItemSelectorCriterion();
+    
+    imageItemSelectorCriterion.setDesiredItemSelectorReturnTypes(
+    desiredItemSelectorReturnTypes);
+    
+    PortletURL itemSelectorURL = `_itemSelector.getItemSelectorURL`(
+    
+      requestBackedPortletURLFactory, "sampleTestSelectItem",
+    
+      imageItemSelectorCriterion);
+
+The return types are declared in an `ArrayList`. The `URLItemSelectorReturnType`
+is added to the list of desired return types, and the 
+`imageItemSelectorCriterion` is set as the kind of element to select. The 
+`ImageItemSelectorCriterion()` constructor is referenced so it can be used. The 
+return types are then set to the item selector criterion using the setter 
+`setDesiredItemSelectorReturnTypes()`. Finally, the item selector URL is created
+using the `getItemSelectorURL()` method, with the value of the JavaScript item 
+selected event name set to `sampleTestSelectItem`.
+
+The order of the item selector criteria is important because it decides the 
+order of the selection views. If you are using one item selector criteria, then 
+this is not an issue for you, but for multiple types of criteria listed in an 
+array this can be an issue. For example, if you pass the item selector criteria 
+in this order: `ImageItemSelectorCriterion`, `VideoItemSelectorCriterion` it 
+will display the selection views for images first and then for videos, but if 
+you reverse the item selector criteria it will display the video selection view 
+first and then the image selection view.
```

Comment:
@sergiogonzalez @robertoDiaz - By "order of the selection views" do you mean the order of the tabs for the selection views?

For example, in the intro image, the selection view for _Documents and Media_ elements is first, followed by the selection view for _URL_ elements?